packages/core/src/tests/factories/documentLogs.ts

```diff
@@ -61,8 +63,11 @@ export async function createDocumentLog({
       return acc + content.length
     }, 0)
     const completionTokens = mockedResponse.length
+    iteration++
+    generatedAt = new Date(generatedAt.getTime() + MS_IN_ONE_DAY * iteration)
```

Comment:
One test where 2 provider logs where created where created with the same timestamp. We add here an artificial day span between logs